lib/rules/sotd/mailing-list.js

```diff
@@ -39,9 +40,15 @@ exports.check = function (sr, done) {
             foundArch = true;
             return;
         }
+        if (/^https:\/\/github.com\/[\w-]+\/[\w-]+/.test(href)) {
```

Comment:
For that rule, it might be safer to look for the issues of the repo (`/^https:\/\/github.com\/[\w-]+\/[\w-]+/issues`). There could be other GH links in the SotD for different purposes.